### PR TITLE
LPS-189460 Objects admin page no longer displays objects after upgrading from U72 to master

### PR DESCRIPTION
**Pull request by seltong**
Related issue: [LPS-189460](https://liferay.atlassian.net/browse/LPS-189460)

### CODE REVIEW

**Review by murilostodolni — CHANGES_REQUESTED** [review]
Hey @seltong 

I finished my review. Let me know if you have questions. Thank you

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:object

**Comment by seltong** [comment]
ci:test:relevant

**Review by murilostodolni — APPROVED** [review]
LGTM 🎉

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v3_8_0/NotificationQueueEntryUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+/**
+ * @author Selton Guedes
+ */
+public class NotificationQueueEntryUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				StringBundler.concat(
+					"select distinct NotificationQueueEntry.",
```
Comment (marcelabc) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v3_8_0/NotificationQueueEntryUpgradeProcess.java:33]: Why did you need to add the **distinct** in the query?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v3_8_0/NotificationQueueEntryUpgradeProcess.java (hunk 1 above)
Comment (seltong) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v3_8_0/NotificationQueueEntryUpgradeProcess.java:33]: In this query we need `notificationTemplateId` and `companyId` to be used to populate the companyId of the NotificationQueueEntry.
We are doing a select in the NotificationQueueEntry table, so we will return all the `notificationTemplateId` and `companyId` of the NotificationQueueEntry instances. But in this case we can have more than one NotificationQueueEntry instaces with the same `notificationTemplateId` and `companyId`. I'm using the distinct to avoid returning duplicate instances in this case.

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by seltong** [comment]
ci:forward
